extensions/wikia/UserLogin/js/UserSignupAjaxValidation.js

```diff
@@ -0,0 +1,118 @@
+/* global  wgScriptPath, UserSignup */
+(function () {
+	'use strict';
+
+	/**
+	 * UserSignupAjaxValidation is contains business logic for ajax signup validation
```

Comment:
Think you have an extra 'is' in there. Should probably be 'UserSignupAjaxValidation contains business logic for ajax signup validation'